exports.getall = async (req, res, next) => {
  try {

  }
  catch (err) {
    console.log(err.message)
  }
}